tests/mpi/eckit_test_mpi.cc

```diff
@@ -1000,6 +1001,8 @@ CASE("test_iProbe") {
         --count;
     }
 
+    std::sort(data.begin(), data.begin() + nproc);
```

Comment:
Given that we know the size is nproc, we may as well write
```c++
std::sort(data.begin(), data.end());
```
or better still
```c++
std::sort(begin(data), end(data));
```

Also, I'd expect the same fix to be also needed for `test_probe`, although I'm too surprised we hit this 
first with `test_iProbe`.

Reply:
Thanks, I have updated as you say.

The failure is transient, so maybe it does affect test_probe as well, I just haven't really noticed.  I have made the same change for test_probe.